introspection-engine/introspection-engine-tests/tests/identify_version/mod.rs

```diff
@@ -215,7 +215,7 @@ async fn introspect_postgres_prisma2(api: &TestApi) -> TestResult {
 
 //Mysql
 
-#[test_connector(tags(Mysql))]
+#[test_connector(tags(Mysql), exclude(TiDB))]
```

Comment:
What is the problem with this test? Is there an issue on TiDB side tracking this?

Reply:
Because TiDB currently does not support the point type, related issue: https://github.com/pingcap/tidb/issues/6347

For this reason, I added an additional test case for TiDB:

https://github.com/prisma/prisma-engines/blob/8177d21bfa7a5699437ae5d4915d946a0ed81f3b/introspection-engine/introspection-engine-tests/tests/identify_version/mod.rs#L352

Reply:
@janpio when doing the prisma documentation probably this can be tagged as an exception ? Any other suggestions ?

Reply:
We do not know yet. Right now a database either supports our feature set, or it does not. We did not have the case where it does, but not a specific type. Depending on the overall story and situation this might be be handled with a comment in the docs and may be a database specific error message - but if there are _more_ things that is probably not enough and we would need to handle this on a higher level (new connector, which explodes our complexity).